src/components/ConfirmRequestDialog.test.tsx

```diff
@@ -0,0 +1,109 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import ConfirmRequestDialog from './ConfirmRequestDialog';
+import DocumentTypesFixture from '../../cypress/fixtures/document-types-response.json';
+import { ResponseMapper } from '../boundary/response-mapper';
+import { EvidenceRequestRequest } from '../gateways/internal-api';
+
+const onAccept = jest.fn();
+const onDismiss = jest.fn();
+const DocumentTypes = DocumentTypesFixture.map(ResponseMapper.mapDocumentType);
+
+describe('Confirm Request Dialog', () => {
+  describe('when there is no request', () => {
+    beforeEach(() => {
+      render(
+        <ConfirmRequestDialog
+          onDismiss={onDismiss}
+          onAccept={onAccept}
+          documentTypes={DocumentTypes}
+        />
+      );
+    });
+
+    it('is closed', () => {
+      expect(screen.queryByRole('dialog')).toBeNull();
+    });
+  });
+
+  describe('when there is a request', () => {
+    const baseRequest: EvidenceRequestRequest = {
+      documentTypes: ['passport'],
+      deliveryMethods: [],
+      resident: {
+        name: 'Frodo',
+        email: 'frodo@bagend.com',
+        phoneNumber: '0123',
+      },
+    };
+
+    beforeEach(() => {
+      render(
+        <ConfirmRequestDialog
+          request={baseRequest}
+          onDismiss={onDismiss}
+          onAccept={onAccept}
+          documentTypes={DocumentTypes}
+        />
+      );
+    });
+
+    it('is closed', () => {
+      expect(screen.queryByRole('dialog')).toBeVisible();
+    });
```

Comment:
The name is "is closed" but it tests the dialog is visible, think one of them needs to change

Reply:
Fixed